zebra/zebra_rib.c

```diff
@@ -829,7 +811,7 @@ struct route_entry *rib_lookup_ipv4(struct prefix_ipv4 *p, vrf_id_t vrf_id)
 
 static unsigned nexthop_active_check(struct route_node *rn,
```

Comment:
Comment above this utility is now stale, with `set` removed?

Reply:
fixed